pkg/runner/runner.go

```diff
@@ -0,0 +1,222 @@
+package runner
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/GoogleContainerTools/kpt/e2e"
+	"github.com/GoogleContainerTools/kpt/run"
+)
+
+// Runner runs an e2e test
+type Runner struct {
+	pkgName  string
+	testCase TestCase
+}
+
+const (
+	expectedDir         string = ".expected"
+	expectedResultsFile string = "results.yaml"
+	expectedDiffFile    string = "diff.patch"
+	expectedConfigFile  string = "config.yaml"
+)
+
+// NewRunner returns a new runner for pkg
+func NewRunner(testCase TestCase) (*Runner, error) {
+	info, err := os.Stat(testCase.Path)
+	if err != nil {
+		return nil, fmt.Errorf("cannot open path %s: %w", testCase.Path, err)
+	}
+	if !info.IsDir() {
+		return nil, fmt.Errorf("path %s is not a directory", testCase.Path)
+	}
+	return &Runner{
+		pkgName:  filepath.Base(testCase.Path),
+		testCase: testCase,
+	}, nil
+}
+
+// Run runs the test.
+func (r *Runner) Run(cmd string, args []string, t *testing.T) error {
+	fmt.Printf("Running test against package %s\n", r.pkgName)
+	tmpDir, err := ioutil.TempDir("", "kpt-fn-catalog-e2e-*")
```

Comment:
`"kpt-fn-catalog-e2e-*"` this should be changed.

Reply:
done